requirements.txt

```diff
@@ -1,5 +1,5 @@
 pypif==2.0.1
 pypif_sdk==2.0.0
-citrination_client
+citrination_client==2.1.0
```

Comment:
The versions in here don't match setup.py (the latter has ranges rather than specific versions). Is that intentional?

Reply:
`requirements.txt` has pinned versions for reproducibility, while `setup.py` has ranges that try to mach the actual support of the underlying libraries.  This is a [standard pattern](https://packaging.python.org/discussions/install-requires-vs-requirements/).